docs/content/examples/tutorials/mailserver-behind-proxy.md

```diff
@@ -72,14 +72,15 @@ Feel free to add your configuration if you achieved the same goal using differen
           - "traefik.tcp.services.smtp.loadbalancer.server.port=25"
           - "traefik.tcp.services.smtp.loadbalancer.proxyProtocol.version=1"
           - "traefik.tcp.routers.smtp-ssl.rule=HostSNI(`*`)"
-          - "traefik.tcp.routers.smtp-ssl.tls=false"
           - "traefik.tcp.routers.smtp-ssl.entrypoints=smtp-ssl"
+          - "traefik.tcp.routers.smtp-ssl.tls.passthrough=true"
           - "traefik.tcp.routers.smtp-ssl.service=smtp-ssl"
           - "traefik.tcp.services.smtp-ssl.loadbalancer.server.port=465"
           - "traefik.tcp.services.smtp-ssl.loadbalancer.proxyProtocol.version=1"
           - "traefik.tcp.routers.imap-ssl.rule=HostSNI(`*`)"
           - "traefik.tcp.routers.imap-ssl.entrypoints=imap-ssl"
           - "traefik.tcp.routers.imap-ssl.service=imap-ssl"
+          - "traefik.tcp.routers.esmtp.tls.passthrough=true"
```

Comment:
I think you meant to reference the `imap-ssl` router not `esmtp`?:

```suggestion
          - "traefik.tcp.routers.imap-ssl.tls.passthrough=true"
```

I'm not familiar with why the `proxyProtocol.version` is sometimes 1 or 2, perhaps due to Postfix / Dovecot support? I understand it's to ensure that the original client IP is handed over correctly so it doesn't get misunderstood as directly from Traefik. But I'm not sure if that's required for each different supported way that Traefik routes to different DMS ports 🤷‍♂️